app/controllers/api/v1/account_controller.rb

```diff
@@ -6,22 +6,14 @@ class Api::V1::AccountController < ApiController
 
   def my_applications
     applications = map_listings_to_applications(current_user_applications)
-    render json: { applications: applications }
+    render json: { applications: }
   end
 
   def update
     contact = account_params
     contact[:contactID] = current_user.salesforce_contact_id
     contact[:webAppID] = current_user.id
-    salesforce_contact = Force::AccountService.create_or_update(
-      webAppID: contact[:id],
-      contactId: contact[:contactID],
-      email: contact[:email],
-      firstName: contact[:firstName],
-      middleName: contact[:middleName],
-      lastName: contact[:lastName],
-      DOB: contact[:DOB],
-    )
+    salesforce_contact = Force::AccountService.create_or_update(contact.as_json)
```

Comment:
This looks good! Just a question: Is there any data from contact that we would not want to pass to the AccountService? I cannot think of any, and I don't think there would be security/privacy concerns since the call to AccountService should be internal.